### DISCUSSION TITLE
有老哥能指导下这个页面怎么做么？

### DISCUSSION THREAD

**Discussion by zzz6519003** [ant-design/ant-design-mobile#4542] [Q&A] [answered]
<img width="236" alt="Screen Shot 2021-12-10 at 9 00 30 PM" src="https://user-images.githubusercontent.com/1625608/145577988-62e4cf61-4634-4659-8680-5bdd45a52889.png">

**Comment by hqwlkj** [comment]
这个是一个页面吗？还是只是从底部弹出的一个动作面板？

**Reply by zzz6519003** [reply]
已经搞定啦 谢谢

**Comment by zzz6519003** [comment, marked as answer]
已经搞定啦 谢谢
